### PR TITLE
Removes TODOs to support VIP180 tokens other than VeThor (#596)

### LINKED ISSUE #596: replace with real symbol for other coins
https://github.com/trustwallet/blockatlas/blob/6f5a514cdedc15f06d6dd2063e721ea514bee59d/platform/vechain/api.go#L148-L153

---

###### This issue was generated by [todo](https://todo.jasonet.co) based on a `TODO` comment in 6f5a514cdedc15f06d6dd2063e721ea514bee59d. It&#x27;s been assigned to @kolya182 because they committed the code.

### PR DESCRIPTION
**Pull request by arein**
Resolves #596 

Also: minor improvement by using the `Coin()` method to resolve coin ID, name, decimals instead of repeatedly hardcoding VET.

### TESTING
* `make test`
* was not able to run functional tests either on my branch or on master

### CODE REVIEW

File: platform/vechain/api_test.go [hunk 1]
```diff
@@ -52,6 +52,10 @@ func TestNormalizeTransaction(t *testing.T) {
 	}{
 		{"Test normalize VET transfer transaction", transferSrc, trxId, expectedTransfer},
 	}
+
+	subject := Platform{}
```
Comment (vikmeup) [inline platform/vechain/api_test.go]:
```suggestion
	platform := Platform{}
```

File: platform/vechain/api_test.go [hunk 2]
```diff
@@ -140,7 +144,7 @@ var expectedTransferLog = blockatlas.TxPage{
 		Status: blockatlas.StatusCompleted,
 		Block:  4382764,
 		Meta: blockatlas.TokenTransfer{
-			Name:     "",
+			Name:     "VeThor",
```
Comment (vikmeup) [inline platform/vechain/api_test.go]:
```suggestion
			Name:     gasTokenName,
```

File: platform/vechain/api_test.go [hunk 3]
```diff
@@ -140,7 +144,7 @@ var expectedTransferLog = blockatlas.TxPage{
...
 			Symbol:   "VTHO",
```
Comment (vikmeup) [inline platform/vechain/api_test.go]:
```suggestion
			Symbol:   gasTokenSymbol,
```

File: platform/vechain/api_test.go [hunk 4]
```diff
@@ -160,6 +164,9 @@ func TestNormalizeTokenTransaction(t *testing.T) {
 	}{
 		{"Normalize VIP180 token transfer", transferLogSrc, trxReceipt, expectedTransferLog},
 	}
+
+	subject := Platform{}
```
Comment (vikmeup) [inline platform/vechain/api_test.go]:
```suggestion
	platform := Platform{}
```

**Comment by arein** [comment]
@catenocrypt @vikmeup thx for looking. Accepted the suggestions. Please merge

**Pull request merged**